releasenotes/notes/allow-mini-agent-to-run-on-all-hosting-plans-5e8df7330c108d71.yaml

```
@@ -0,0 +1,4 @@
+
+features:
+  - |
+    tracing: This removes the restrictions on the tracer to only run the mini-agent on the consumption plan. The mini-agent now runs regardless of the hosting plan
```

Add new line and nits
```suggestion
    tracing: Removes the restrictions on the tracer to only run the mini-agent on the consumption plan. The mini-agent now runs regardless of the hosting plan.

```